state/a_inputpassword.cpp

```diff
@@ -138,8 +138,7 @@ void InputPassword::onLoginResult(bool ok) {
 
             if ( config::isOnlineWallet() && context->wallet->hasPassword() ) {
                 // Start listening, no feedback interested
-                context->wallet->listeningStart(true, false, true);
-                context->wallet->listeningStart(false, true, true);
+                context->wallet->listeningStart(context->appContext->isAutoStartMQSEnabled(), context->appContext->isAutoStartKeybaseEnabled(), true);
```

Comment:
That will not work for ,true, true, case.  Sorry, interface is confusing, single call can start only one listener.
will work:
if (context->appContext->isAutoStartMQSEnabled())
     context->wallet->listeningStart(true, false, true);
if (context->appContext->isAutoStartKeybaseEnabled())
    context->wallet->listeningStart(false, true, true);